t04_refactored.py

```diff
@@ -158,9 +158,66 @@ def team_7_adv():
 
 ###################################################################################
 
+
 def team_8_adv():
     pass
```

Comment:
Missing the docstring. Check the Google doc for instructions on the proper contents for this assignment.